js/autotrack/common.ts

```diff
@@ -54,12 +96,10 @@ const getComponentHierarchyTraversal = componentThis => {
 };
 
 // Traverse up the hierarchy from the current component up to the root, and return an array of
-// objects representing the component hierarchy from root to the current node. Each object element
-// contains:
-// * elementName - The name of the component
-// * fiberNode - The FiberNode reference for the component
-// * propsString - The string of props obtained from extractProps().
-const getFiberNodeComponentHierarchyTraversal = currNode => {
+// objects representing the component hierarchy from root to the current node.
+const getFiberNodeComponentHierarchyTraversal: (
+  currNode: FiberNode | null
```

Comment:
Hmm.... looking at line 104 below, it seems like the return type should be `FiberNode[]` ?  That way it _could_ be empty, right?

Reply:
Not sure what you mean.  `currNode.return` (the argument to the recursive call) is of type `FiberNode | null`, and the resultant hierarchy traversal of a given `FiberNode | null` is of type `ComponentHierarchyTraversalElement[]`